### DISCUSSION TITLE
Simplifying schematics

### DISCUSSION THREAD

**Discussion by muxa** [issalig/toshiba_air_cond#24] [General]
Ive removed one of the two transistor from the write circuit, as discussed here: https://github.com/issalig/toshiba_air_cond/discussions/22

I'd like to see if circuits can be simplified further. 

Referring to https://github.com/issalig/toshiba_air_cond/blob/master/pcb/airpcb_smd/schematics.pdf

* For instance, removing D2 and increasing the R3 value. What is the role of D2 apart from dropping voltage?
* In addition, replacing D4 diode with a resistor. In the circuijs.txt model there's a resistor instead of a diode. What is the role of D4?

**Comment by issalig** [comment]
- Yes, you are true, it is smarter to use a resistor in order to drop the voltage instead of D2.
- For D4 now, I do not remember the reason. Looking at the write circuit now, it look strange to me. Probably it was just to assure A is higher than B.

Sorry, I did the circuit 2 years ago and once it worked I forgot the reasons. (Also, I am not an EE so the circuits are not optimal neither logic :) )

Anyway, thanks for collaborating and tell me if these modifications work ok.
